File: TicketReservation.py
#definitions and declarations
class customer:
    username = ""
    phone = 00000000000
    enrollment = 0


class tickets:
    ticketnumber = 0
    bookingname = ""


ticketsarray = []
customersarray = []

def bookticket(username):
    t1 = tickets()
    t1.bookingname = username
    t1.ticketnumber += 1
    ticketsarray.append(t1)
    print ("Ticket Booked.")

def cancelbooking(username):
    for ticket in ticketsarray:
        if ticket.bookingname == username:
            ticketsarray.remove(ticket)
            print ("Ticket Cancelled")
        else:
            print ("Ticket Does not Exist")

while(1):
    print ("====================================================")
    print ("Welcome to tickets reservation system")
    print ("Enter:")
    print ("1. Book A Ticket")
    print ("2. Cancel A Booked Ticket")
    print ("3. See Booked Tickets")
    print ("4. Exit")
    choice = int(input())
    if choice == 1:
        print ("Enter your username:")
        username = input()
        print ("Enter your phone number:")
        phone = input()
        c1 = customer()
        c1.username = username
        c1.phone = phone
        c1.enrollment+=1
        customersarray.append(c1)
        bookticket(c1.username)
    elif choice == 2:
        print ("Enter your username:")
        username = input()
        cancelbooking(username)
        for customer in customersarray:
            if customer.username == username:
                customersarray.remove(customer)
    elif choice == 3:
            if len(ticketsarray) != 0:
                print ("Tickets booked: ", len(ticketsarray))
                for ticket in ticketsarray:
                    print ("Username: ", ticket.bookingname)
                    print ("Ticket Number: ", ticket.ticketnumber)
            else:
                print ("Zero tickets booked yet")
    elif choice == 4:
        break

File: TicketReservation.cs
using System;
using System.Collections.Generic;

public class Program
{

    public static void Main()
    {
        IDictionary<string, string> ticketlist = new Dictionary<string, string>();

        while (true)
        {
            System.Console.WriteLine("=======================================");
            System.Console.WriteLine("Welcome to Ticket Reservation System:");
            System.Console.WriteLine("Enter:");
            System.Console.WriteLine("1. Book A ticket");
            System.Console.WriteLine("2. Cancel A Booking");
            System.Console.WriteLine("3. See The booked Tickets:");
            System.Console.WriteLine("4. Exit");
            String choice = Console.ReadLine();
            if (choice == "1")
            {
                customer c1 = new customer();
                tickets t1 = new tickets();
                System.Console.WriteLine("Enter Your Username:");
                c1.username = Console.ReadLine();
                System.Console.WriteLine("Enter Your Phone Number:");
                c1.phone = Console.ReadLine();
                c1.enrollment++;
                ticketlist.Add(c1.username, (t1.ticketnumber).ToString());
                System.Console.WriteLine("Ticket Booked: " + ticketlist.Count);
            }
            else if (choice == "2")
            {
                System.Console.WriteLine("Enter Your Username:");
                string username = Console.ReadLine();
                ticketlist.Remove(username);
                System.Console.WriteLine("Ticket Cancelled");
            }
            else if (choice == "3")
            {
                System.Console.WriteLine("Tickets Booked In Total: " + ticketlist.Count);
            }
            else if (choice == "4")
            {
                break;
            }
            else
            {
                System.Console.WriteLine("Wrong Input Entered");
            }

        }
    }
}

public class customer
{

    public string username;
    public string phone;
    public int enrollment = 0;
}

public class tickets
{

    public int ticketnumber;
    public string bookingname;
}